### PR TITLE
Change Version and add readme.md

### PR DESCRIPTION
**Pull request by Jimboi2K**
Add a ReadMe file that links to the website.

Changed version for the new items/economy update

### CODE REVIEW

**Comment by Jimboi2K** [comment]
Also I added a linter (prettier, default) to lint files

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This is a basic workflow that is manually triggered
+
+name: Linter
```
Comment (Jimboi2K) [inline .github/workflows/lint.yml:3]: Will update default notices

**Review by Jimboi2K — COMMENTED** [review]
Just need to remove the default messages in the file, and replace them with descriptive things

File: .github/workflows/lint.yml (hunk 1 above)
Comment (CoolJim) [reply .github/workflows/lint.yml:3]: OK I will do that actually, post pr merge

**Pull request merged**